lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -422,6 +423,17 @@ def self.available_options
                                   self.optional_error("Extension not supported: '#{file_ext}'. Supported formats for platform '#{platform}': #{accepted_formats.join ' '}") unless accepted_formats.include? file_ext
                                 end
                               end),
+          
+          FastlaneCore::ConfigItem.new(key: :upload_build_only,
+                                  env_name: "APPCENTER_DISTRIBUTE_UPLOAD_BUILD_ONLY",
+                               description: "Flag to upload only the build to App Center. Skips uploading symbols or mapping",
+                                  optional: true,
+                                 is_string: false,
+                             default_value: false,
+                       conflicting_options: [:upload_dsym_only, :upload_mapping_only],
+                            conflict_block: proc do |value|
+                              UI.user_error!("You can't use 'upload_build_only' and '#{value.key}' options in one run")
```

Comment:
Interestingly, this is only true if the flag was set to `true`. If we explicitly set to `false` then the options are somewhat compatible.

Reply:
oh yeah. nice catch. will remove this conflict block.. because we can't check the values set to true/false.

Reply:
As discussed, we are going with this as it won't make sense to provide those fields at same time, even if its true/false